Flaky test: TestDaemonRestartKillContainers

Seen failing on at least ppc64le and s390x
https://jenkins.dockerproject.org/job/Docker-PRs-s390x/7514/consoleFull

```
15:16:39 --- FAIL: TestDaemonRestartKillContainers (0.00s)
...
15:16:39     --- FAIL: TestDaemonRestartKillContainers/live-restore=false/container_with_restart=always/stop-daemon (0.25s)
15:16:39     	daemon.go:285: [daeec8d199488] waiting for daemon to start
15:16:39     	daemon.go:275: [daeec8d199488] exiting daemon
15:16:39     	daemon.go:203: Error starting daemon with arguments: []
```

--- a/integration/container/restart_test.go
+++ b/integration/container/restart_test.go
@@ -59,9 +59,9 @@ func TestDaemonRestartKillContainers(t *testing.T) {
 						t.Fatal(err)
 					}
 
-					var args []string
+					args := []string{"--iptables=false"}
 					if liveRestoreEnabled {
-						args = []string{"--live-restore"}
+						args = append(args, "--live-restore")
 					}
 
 					d.StartWithBusybox(t, args...)

